components/builder-web/app/package/package-sidebar/_package-sidebar.component.scss

```diff
@@ -9,5 +9,10 @@
     hab-icon {
       float: right;
     }
+
+    .wbs { // for wee bit smaller
+      font-size: smaller;
```

Comment:
just looking around and it seems the standard used in this code base is t ouse precise pixel sizes. There seem to be some helper variables defined for this in _base.scss. For example `$small-font-size`. So it might be worth considering one of those variables over `smaller`.

Reply:
I dug back into this and experimented.  I went with smaller because it was relative sizing instead of absolute and while I have limited experience implementing responsive layouts that felt better to me based on what I think I know in a responsive design situation.  I also studied the sass bits you pointed out and looked at things with Firefox's developer tools. Using smaller the font size becomes 13.333px.  I could break towards explicitly specifying 13px but given my experiments and research today I think I'd like to maintain this sizing ratio if something else changed in the future. Unless you feel strongly about it, my inclination is to keep it as is at this point.

Reply:
I'm not incredibly concerned but it seemed worth calling out that absolute sizes are used everywhere else. I'll leave this up to you.